docs/UpgradeGuide.md

```diff
@@ -149,10 +149,14 @@ Check out the table below for a list of common mappings:
 | `tracer.transport_options(options)`    | Removed                       |
 | `tracer.writer`                        | `tracing.writer`              |
 | `tracer.writer_options`                | `tracing.writer_options`      |
+| `use`                                  | `tracing.instrument`          |
```

Comment:
IIUC this is true for tracing integrations but would be `ci.instrument` for CI ones, correct?

Reply:
Yes, this is true.